presto-main/src/test/java/com/facebook/presto/operator/scalar/TestArrayIntersectFunction.java

```diff
@@ -215,6 +215,9 @@ public void testDuplicates()
     public void testSqlFunctions()
     {
         assertFunction("array_intersect(ARRAY[ARRAY[1, 3, 5], ARRAY[2, 3, 5], ARRAY[3, 3, 3, 6]])", new ArrayType(INTEGER), ImmutableList.of(3));
+        assertFunction("array_intersect(ARRAY[null, ARRAY[], ARRAY[1, 2, 3]])", new ArrayType(INTEGER), null);
+        assertFunction("array_intersect(ARRAY[ARRAY[], null, ARRAY[1, 2, 3]])", new ArrayType(INTEGER), null);
+        assertFunction("array_intersect(ARRAY[])", new ArrayType(UNKNOWN), ImmutableList.of());
```

Comment:
@kewang1024 
I don't see a test case for 
`array_intersect(null)
`
Let's add it?